server/src/app.ts

```diff
@@ -4,6 +4,7 @@ import { v4 as uuidv4 } from "uuid";
 const app = express();
 const port = 3000;
 const cors = require("cors");
+const md5 = require("md5");
```

Comment:
Don't use md5. It's extremely unsafe and should NEVER be used. It was cracked and a lot of rainbow tables are online for md5. SHA-1 used to be a good replacement but it's officially considered vulnerable too. I suggest using SHA-256, SHA-384 or SHA-512. These are safe and will work easily with JS.